src/components/MenuBars/MenuItem/MenuItem.scss

```diff
@@ -15,6 +11,7 @@ $icon-background-color-dark: #333948;
   padding: 0;
   position: relative;
   cursor: pointer;
+  pointer-events: auto;
```

Comment:
isn't it `auto` by default?

Reply:
No, somehow the element inherited `none` from the container. But I have changed the grid container anyways, so `pointer-events` is gone :)